regex-syntax/src/hir/mod.rs

```diff
@@ -2134,6 +2135,17 @@ impl Properties {
         self.0.look_set_suffix_any
     }
 
+    /// Returns whether there are any look-around expressions in this HIR value.
+    ///
+    /// Only returns true for [`HirKind::LookAround`] and not for
+    /// [`HirKind::Look`], which can be queried by [`look_set`] instead.
+    /// Currently, only lookbehind assertions without capture groups are
+    /// supported.
```

Comment:
I would not mention this here. This is not the concern of `Properties` and will make it harder to update the docs when the support is changed.